pinot-query-planner/src/main/java/org/apache/pinot/query/planner/partitioning/HashFunctionSelector.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.query.planner.partitioning;
+
+import java.nio.charset.StandardCharsets;
+import org.apache.pinot.spi.utils.hash.CityHashFunctions;
+import org.apache.pinot.spi.utils.hash.MurmurHashFunctions;
+
+
+/**
+ * Utility class to compute hash values using different hash functions.
+ * This class provides consistent hash computation for KeySelector implementations.
+ */
+public class HashFunctionSelector {
+  public static final String MURMUR2 = "murmur2";
```

Comment:
Can you verify with the Quickstart what's the function name stored in HashDistributionDesc in the Colocated Join quickstart (you can debug around `attemptPartitionedDistribution`)